packages/slate-base64-serializer/src/index.js

```diff
@@ -1,5 +1,49 @@
 
 import { State } from 'slate'
+import { Buffer } from 'buffer'
+
+function newBuffer(data, encoding, len) {
+  return new Buffer(data, encoding, len)
+}
+
+if (!Buffer.from) {
+  Buffer.from = newBuffer
```

Comment:
Monkey-patching anything could potentially be bad

Reply:
this is only necessary for node before v4 as Buffer.from is only available from v4 onwards

Reply:
Yes, but you are extending the Buffer object and people might run into edge cases. Imagine people are checking for Buffer.from somewhere else to see if they are running an old version of node.

Reply:
fair point, I'll remove the monkey patching and use it without monkey patching